backend/cmd/headlamp.go

```diff
@@ -365,21 +365,23 @@ func createHeadlampHandler(config *HeadlampConfig) http.Handler {
 	fmt.Println("*** Headlamp Server ***")
 	fmt.Println("  API Routers:")
 
-	// load kubeConfig clusters
-	err := kubeconfig.LoadAndStoreKubeConfigs(config.kubeConfigStore, kubeConfigPath, kubeconfig.KubeConfig)
-	if err != nil {
-		logger.Log(logger.LevelError, nil, err, "loading kubeconfig")
-	}
+	if !config.useInCluster {
```

Comment:
Many users have asked how they could add more clusters when headlamp is running in a cluster. I find the mixed use-case a bit cornerish, but don't see a reason why we should restrict mixing loading kubeconfigs with in-cluster. So maybe what we need to do is not log issues about loading kubeconfigs if we are running in-cluster.